modules/ROOT/pages/6.6.1-release-notes.adoc

```diff
@@ -49,10 +49,20 @@ The {productname} 6.6.1 release includes an accompanying release of the **AI Ass
 // CCFR here.
 
 
-==== Using AI Assistant within OSX Safari, caused the editor to scroll to the bottom of the editor
+==== When Safari on macOS was the host environment and a returned response was inserted into the {productname} editor, the inserted content did not scroll into view
 //#TINY-10102
 
-// CCFR here.
+Previously when:
+
+. Safari running on macOS was the host browser; and
+. the **AI Assistant** dialog was open and contained a returned response; and
+. the *Insert* button was pressed
+
+the **AI Assistant** dialog closed but, if the inserted content was not visible, the {productname} editor instance did not scroll the returned response into view.
+
+**AI Assistant** 1.0.1 fixes this.
+
+With this release, after the *Insert* button is pressed and the **AI Assistant** dialog closes, if the inserted response is not visible, the {productname} editor scrolls it into view.
```

Comment:
```suggestion
Previously when inserting a response from the AI Assistant dialog at a selection which is above the current view, the inserted content would not be scrolled into view.

In **AI Assistant** 1.0.1, the response is now inserted after the dialog is closed, allowing the content to be scrolled into view correctly.
```
It wasn't a browser specific issue, just the order of operations from an incomplete workaround.